src/dialogs/config-flow/show-dialog-config-flow.ts

```diff
@@ -73,7 +73,12 @@ export const showConfigFlowDialog = (
       );
       return description
         ? html`
-            <ha-markdown allow-svg breaks .content=${description}></ha-markdown>
+            <ha-markdown
+              allow-data-url
```

Comment:
We can limit it to only ZwaveJS like this
```suggestion
              .allowDataUrl=${step.handler === "zwave_js"}
```

Reply:
Let's limit it to zwave, if we have more need for it we can reconsider later